.github/workflows/test.yml

```diff
@@ -26,10 +26,6 @@ jobs:
       - name: Lint
         run: npm run lint
 
-      - name: Verify dependencies [server-renderer]
```

Comment:
This is no longer needed, `npm ci` natively understands workspaces and will install everything from the root

Reply:
> This is no longer needed, `npm ci` natively understands workspaces and will install everything from the root

Hi James, we actually have 2 repos and the `renderer` one is the subfolder of root but not a module. So we need run ci seperated for them.

Reply:
Hey @Koooooo-7, can you describe the issue here more? 

What James is mentioning is that the above step,

```
        run: npm ci --ignore-scripts
```

also installs dependencies for server-renderer.

Reply:
I think we're fine: I don't think many people (if any?) are using SSR, and our current `dev:ssr` script is even broken like so:

```js
$ npm run dev:ssr
/Users/trusktr/src/docsifyjs+docsify/packages/docsify-server-renderer/build.js:8
var fetch = _interopDefault(require('node-fetch'));
                            ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/trusktr/src/docsifyjs+docsify/packages/docsify-server-renderer/node_modules/node-fetch/src/index.js from /Users/trusktr/src/docsifyjs+docsify/packages/docsify-server-renderer/build.js not supported.
Instead change the require of index.js in /Users/trusktr/src/lume+lume+develop/packages/docsifyjs+docsify/packages/docsify-server-renderer/build.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/trusktr/src/lume+lume+develop/packages/docsifyjs+docsify/packages/docsify-server-renderer/build.js:8:29)
    at Object.<anonymous> (/Users/trusktr/src/lume+lume+develop/packages/docsifyjs+docsify/server.js:6:20) {
  code: 'ERR_REQUIRE_ESM'
}

Node.js v18.13.0
lib/plugins/emoji.js
ERROR: "serve:ssr" exited with 1.
```

I a new issue which should address this: https://github.com/docsifyjs/docsify/issues/2102

I think it would be ok to merge this then fix any issues we may find, if we even care about them.

Even better: I recommend we just delete the SSR stuff because it is a pain to maintain that none of us have time for. In fact, install Docsify plugins and SSR breaks very easily. It is in a very alpha quality that we aren't using ourselves and honestly haven't cared for, so I think deleting SSR, and improving our code base (move to ESM, clean up some logic), would help us better prepare for thinking about the proper way to implement SSR.

If we decide to then restore SSR, I think we should convert to a `Docsify` component (f.e. using Solid.js, React, Vue, Svelte, or etc) and let a tool like Astro (maintained by a team of people who think about SSR and SSG all day every day, and are making the latest advancements in the area) give us the best SSR and SSG setup that we can simply ride upon. This will be very powerful, and without us having our own SSR implementation that doesn't work well and is far from complete.

With this in mind, I would be ok to roll with this, as I tested non-SSR Docsify and the dev experience is working great.